Makefile

```diff
@@ -108,6 +101,13 @@ coverage: test
 	cat $(COVERAGE_FILE) | grep -v "_mock.go" > $(COVERAGE_FILE).no-mocks
 	go tool cover -func=$(COVERAGE_FILE).no-mocks
 
+ginkgo:
```

Comment:
Let's move this to `tests/e2e/Makefile` instead.

Reply:
k will do

Reply:
done